### PR TITLE
Don't use docker.io images because of the rate limit

### PR DESCRIPTION
**Pull request by droslean**
https://issues.redhat.com/browse/DPTP-2226

/hold

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

**Comment by droslean** [comment]
/cc @openshift/openshift-team-developer-productivity-test-platform 

Holding to investigate solutions for the rest of the images. 

@alvaroaleman Can you add `vault-subpath-proxy:latest` to your quay.io as well?

**Comment by droslean** [comment]
Also, I still feel that it is better to just create imageStreams that mirror the dockerhub images.

File: clusters/app.ci/vault/manifests.yaml [hunk 1]
```diff
@@ -186,7 +186,7 @@ spec:
           preStop:
             exec:
               command: ["/bin/sleep", "30"]
-      - image: vault:1.7.0
```
Comment (alvaroaleman) [inline clusters/app.ci/vault/manifests.yaml:189]: This comes from an imagestream in the cluster
/hold

Comment (droslean) [reply clusters/app.ci/vault/manifests.yaml:189]: From where?

Comment (droslean) [reply clusters/app.ci/vault/manifests.yaml:189]: The imagestream is there. Do we have also for `subpath-proxy` ?

Comment (alvaroaleman) [reply clusters/app.ci/vault/manifests.yaml:189]: It's in the very same file: https://github.com/openshift/release/blob/dfc0e6ac61aaded5799237e650dcc1ab38ba81fb/clusters/app.ci/vault/manifests.yaml#L30-L52

Subpath proxy is built from our own code so doesn't need anything

Comment (droslean) [reply clusters/app.ci/vault/manifests.yaml:189]: Great.. I will remove vault from my list then :) Thanks.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**